recipe/meta.yaml

```diff
@@ -64,11 +65,15 @@ test:
     {% set tests_to_skip = tests_to_skip + " or test_fsspec_memory_list" %}
     {% set tests_to_skip = tests_to_skip + " or test_elastic_training_dl1_backend_gloo" %}
     {% set tests_to_skip = tests_to_skip + " or test_elastic_training_dl2_backend_gloo" %}
+    # fails because fsspec is not available (AWS S3 stuff)
+    {% set tests_to_skip = tests_to_skip + " or test_fsspec_io_iterdatapipe" %}
+    {% set tests_to_skip = tests_to_skip + " or test_s3_io_iterdatapipe" %}
     # tend to fail due to Google Drive rate-limiting
     {% set tests_to_skip = tests_to_skip + " or test_gdrive_iterdatapipe" %}
     {% set tests_to_skip = tests_to_skip + " or test_online_iterdatapipe" %}
-    # unclear this fails only on py<=39
-    {% set tests_to_skip = tests_to_skip + " or test_fsspec_io_iterdatapipe" %}  # [py<=39]
+    # 20231124 - disable tests that might timeout after 6 hours
+    # https://github.com/pytorch/data/blob/v0.7.1/test/dataloader2/test_mprs.py#L233
+    {% set tests_to_skip = tests_to_skip + " or test_early_exit_ctx_" %}
```

Comment:
Disabling these multiprocessing related tests from https://github.com/pytorch/data/blob/v0.7.1/test/dataloader2/test_mprs.py#L233 because they can lead to timeouts after 6 hours (see previous failure at commit 8df39110c1c352568ac7df3c2f8c3f0bc282c2bd, e.g. https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=823046&view=logs&j=4f922444-fdfe-5dcf-b824-02f86439ef14&t=937c195f-508d-5135-dc9f-d4b5730df0f7&l=1080)